.travis.yml

```diff
@@ -1,5 +1,9 @@
+dist: trusty
+sudo: required
 language: python
-python: 2.7
+python:
+  - 2.7
+  - 3.6
```

Comment:
We do not support Python 3.6 yet at this stage since ScanCode does not support it for now

Reply:
Yeah keeping that in mind I added these two versions, as coala will need python3+ versions. You can see in this file that I used conditional logic to run ScanCode in python2.7 and coala in python3.6

Reply:
Cannot you just run coala in the same build run? e.g. python3 is likely available in both cases.

Reply:
I tried running it in same build but travis failed each time, here are the failed logs:
https://travis-ci.org/nexB/scancode-server/builds/288168067
https://travis-ci.org/nexB/scancode-server/builds/288178128
https://travis-ci.org/nexB/scancode-server/builds/289554875
I'll give it a try running it in the same build, if you wish.

Reply:
It is usually better to run the linters in a separate job, so that they run concurrently with the unit tests

You can use `matrix: include: ` to add an extra job for `coala-ci`.  The matrix entry can have its own `script:` etc, so you do not need to modify the main job.